release/create_release_archives.sh

```diff
@@ -139,6 +139,13 @@ find install/tools -type f -exec "${CP}" --parents {} "${COMMON_FILES_DIR}" \;
 find tools -type f -not -name "githubContrib*" -not -name ".*" -exec "${CP}" --parents {} "${COMMON_FILES_DIR}" \;
 popd
 
+ls -l  "${COMMON_FILES_DIR}/install/kubernetes/"
+ls -l  "${COMMON_FILES_DIR}/install/kubernetes/helm/istio"
+
+# merge values-istio-demo-common.yaml into values-istio-demo yaml files
+cat"${COMMON_FILES_DIR}/install/kubernetes/helm/istio/values-istio-demo-common.yaml" >> "${COMMON_FILES_DIR}/install/kubernetes/helm/istio/values-istio-demo.yaml"
```

Comment:
missing a space after `cat`

Reply:
yes corrected, thanks.

@mandarjog I also changed the file to insert common file first.